application/src/main/java/org/togetherjava/tjbot/commands/Features.java

```diff
@@ -52,6 +57,7 @@
         TagSystem tagSystem = new TagSystem(database);
         ModerationActionsStore actionsStore = new ModerationActionsStore(database);
         ModAuditLogWriter modAuditLogWriter = new ModAuditLogWriter(config);
+        ChannelMonitor channelMonitor = new ChannelMonitor();
```

Comment:
if this class escapes its package now and goes into such a prominent place, it has to be renamed so that its context is clear. `FreeChannelMonitor` for example.